packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -101,10 +101,16 @@ export function getMethod(action) {
 
 export function mergeConfiguredHeader(config) {
 	// still need to keep the previous header added by action
-	return options => ({
-		...options,
-		headers: { ...options.headers, ...DEFAULT_HTTP_HEADERS, ...config.headers },
-	});
+	return options => {
+		const headerMergedConfig = {
+			...options,
+			headers: { ...DEFAULT_HTTP_HEADERS, ...config.headers, ...options.headers },
+		};
+		if (headerMergedConfig.body && headerMergedConfig.body instanceof FormData) {
```

Comment:
why the first condition is required ? If no .body it will not be an instance of FormData right.

Reply:
right changing